chain.go

```diff
@@ -134,21 +134,20 @@ func (taskChain *TaskChain) Run() error {
 }
 
 // formatDuration is a time formatter.
-func formatDuration(d time.Duration) (s string) {
-	timeMsg := ""
+func formatDuration(d time.Duration) (timeMsg string) {
 
 	if d.Hours() > 1.0 {
-		timeMsg += fmt.Sprintf("%1.0fh", d.Hours())
+		timeMsg = fmt.Sprintf("%1.0fh", d.Hours())
 	}
 
 	if d.Minutes() > 1.0 {
-		timeMsg += fmt.Sprintf("%1.0fm", d.Minutes())
```

Comment:
Concatenation was unnecessary only for hours. Because initially we had blank string. Now we always rewrite string variable which may not be the intention. Sorry for confusing you with unclear comment.